hack/verify-golint.sh

```diff
@@ -74,7 +74,7 @@ for p in "${all_packages[@]}"; do
 	# and not just the ones for the current platform.
 	# Packages with a corresponding foo_test package will make golint fail
 	# with a useless error. Just ignore that, see golang/lint#68.
-	failedLint=$(golint "$p"/*.go 2>/dev/null)
+	failedLint=$(ls "$p"/*.go | egrep -v "(zz_generated.*.go|generated.pb.go|generated.proto|types_swagger_doc_generated.go)" | xargs -L1 golint 2>/dev/null)
```

Comment:
@cblecker thank you!

This line looks good. If CI likes it overall, lgtm. :+1: